flake.lock

```diff
@@ -0,0 +1,234 @@
+      "original": {
+        "owner": "nix-community",
+        "repo": "nixpkgs.lib",
+        "type": "github"
+      }
+    },
+    "nixpkgs_2": {
+      "locked": {
+        "lastModified": 1744868846,
+        "narHash": "sha256-5RJTdUHDmj12Qsv7XOhuospjAjATNiTMElplWnJE9Hs=",
+        "owner": "NixOS",
+        "repo": "nixpkgs",
+        "rev": "ebe4301cbd8f81c4f8d3244b3632338bbeb6d49c",
+        "type": "github"
+      },
+      "original": {
+        "owner": "NixOS",
+        "ref": "nixpkgs-unstable",
+        "repo": "nixpkgs",
+        "type": "github"
+      }
+    },
+    "nixpkgs_3": {
+      "locked": {
+        "lastModified": 1744868846,
+        "narHash": "sha256-5RJTdUHDmj12Qsv7XOhuospjAjATNiTMElplWnJE9Hs=",
+        "owner": "NixOS",
+        "repo": "nixpkgs",
+        "rev": "ebe4301cbd8f81c4f8d3244b3632338bbeb6d49c",
+        "type": "github"
+      },
+      "original": {
+        "owner": "NixOS",
+        "ref": "nixpkgs-unstable",
+        "repo": "nixpkgs",
+        "type": "github"
+      }
+    },
+    "nixpkgs_4": {
+      "locked": {
+        "lastModified": 1744868846,
+        "narHash": "sha256-5RJTdUHDmj12Qsv7XOhuospjAjATNiTMElplWnJE9Hs=",
+        "owner": "NixOS",
+        "repo": "nixpkgs",
+        "rev": "ebe4301cbd8f81c4f8d3244b3632338bbeb6d49c",
+        "type": "github"
+      },
+      "original": {
+        "owner": "NixOS",
+        "ref": "nixpkgs-unstable",
+        "repo": "nixpkgs",
+        "type": "github"
+      }
+    },
+    "qgisLatest": {
+      "inputs": {
+        "flake-parts": "flake-parts",
+        "nixpkgs": "nixpkgs_2"
+      },
+      "locked": {
+        "lastModified": 1746268773,
+        "narHash": "sha256-WO1SHo96bKcPkZ0s9GogL0JqEukhwTZrSsw+bwl79m4=",
+        "owner": "imincik",
+        "repo": "QGIS",
+        "rev": "0e95bafe8cdc920080c53b7d15ab867eb78f631d",
+        "type": "github"
+      },
+      "original": {
+        "owner": "imincik",
+        "ref": "nix-package-release-3_42",
+        "repo": "QGIS",
+        "type": "github"
+      }
+    },
+    "qgisLtr": {
+      "inputs": {
+        "flake-parts": "flake-parts_2",
+        "nixpkgs": "nixpkgs_3"
+      },
+      "locked": {
+        "lastModified": 1746604554,
+        "narHash": "sha256-uglIhnYAN7dB02sBr+DV2KlYtARURUVNuP0uIsiEAs8=",
+        "owner": "imincik",
+        "repo": "QGIS",
+        "rev": "e82dcace180741f5e23baea8c46b08d625fba14f",
+        "type": "github"
+      },
+      "original": {
+        "owner": "imincik",
+        "ref": "nix-package-release-3_40",
+        "repo": "QGIS",
+        "type": "github"
+      }
+    },
+    "qgisMaster": {
+      "inputs": {
+        "flake-parts": "flake-parts_3",
+        "nixpkgs": "nixpkgs_4"
+      },
+      "locked": {
+        "lastModified": 1746545503,
+        "narHash": "sha256-3bBzsn+6swpEKk5obmENeQj4XmMkxXFdze9oQ0xfdIs=",
+        "owner": "imincik",
+        "repo": "QGIS",
+        "rev": "43f75f91b5d16a8bd83cee030e015839cc2b76c2",
```

Comment:
This looks odd to me -- why are we using a fork here? And will this always pin the "master" docs to some old commit?

Reply:
Fork is was used here until Nix packaging was merged to QGIS. Now Nix packaging is merged to master branch and fork was removed (for this branch).